src/Products/Annotation/Controllers/AnnotationApiController.cs

```diff
@@ -592,56 +504,30 @@ public HttpResponseMessage Annotate(AnnotationPostedDataEntity annotateDocumentR
             return Request.CreateResponse(HttpStatusCode.OK, annotatedDocument);
         }
 
-        /// <summary>
-        /// Get all annotations from the document
-        /// </summary>
-        /// <param name="documentGuid">string</param>
-        /// <param name="documentType">string</param>
-        /// <returns>AnnotationInfo[]</returns>
-        private AnnotationInfo[] GetAnnotations(string documentGuid, string documentType, string password)
-        {
-            try
-            {
-                FileStream documentStream = new FileStream(documentGuid, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
-                DocumentType docType = DocumentTypesConverter.GetDocumentType(documentType);
-                AnnotationInfo[] annotations = new BaseImporter(documentStream, AnnotationImageHandler, password).ImportAnnotations(docType);
-                documentStream.Dispose();
-                documentStream.Close();
-                return annotations;
-            }
-            catch (System.Exception ex)
-            {
-                throw ex;
-            }
-        }
-
-        private List<PageDataDescriptionEntity> GetAnnotatedPagesForPrint(string path)
+        private List<PageDataDescriptionEntity> GetAnnotatedPagesForPrint(string password, string documentGuid)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'GetAnnotatedPagesForPrint' a static method.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372471)